src/types/PointDetailsType.ts

```diff
@@ -0,0 +1,28 @@
+// Response returned by azimuth#getPoint
```

Comment:
wondering what's the best way to handle this for the layer-2 types... 

[`roller.getPoint`](https://urbit.github.io/roller-rpc-client/index.html#getpoint) returns this type: https://urbit.github.io/roller-rpc-client/interfaces/point.html)

Reply:
Good question. One idea is to take advantage of Typescript [namespaces](https://www.typescriptlang.org/docs/handbook/namespaces.html); alternatively, we can go with something simpler like naming them explicitly (e.g., `L1Point`, `L2Point`). It's less complex and quicker to go with the latter solution, but potentially more maintainable in the long run to use namespaces. WDYT?

Reply:
I like simple :D I'll update the Roller to return an L2Point